File: products/Electrodomestico.java
package products;

public class Electrodomestico extends Product{
    double energyUsed;

    public Electrodomestico(String name, String importDate, double price, int stock, double energyUsed) {
        super(name, importDate, price, stock);
        this.energyUsed = energyUsed;
    }

    public double getEnergyUsed() {
        return energyUsed;
    }
}

File: electrodomestico.py
from product import Product

class Electrodomestico(Product):
    def __init__(self, name, import_date, price, stock, energy_used):
        super().__init__(name, import_date, price, stock)
        self.energyUsed = energy_used

    def getEnergyUsed(self):
        return self.energyUsed